src/Lean/Util/Paths.lean

```diff
@@ -14,8 +14,9 @@ namespace Lean
 open System
 
 structure LeanPaths where
-  oleanPath : SearchPath
-  srcPath   : SearchPath
+  oleanPath       : SearchPath
+  srcPath         : SearchPath
+  loadDynlibPaths : Array FilePath := #[]
```

Comment:
Why not `loadDynlibPath : SearchPath := []` for consistency?

Reply:
That doesn't seem semantically appropriate, it's not a search path in any way. We simply load all of them.

Reply:
Ah, true, makes sense.